### PR TITLE
Jquery in view fork

### PR DESCRIPTION
**Pull request by ricardopolo**
This fork is more popular than the original project

### CODE REVIEW

**Comment by ricardopolo** [comment]
@jimaek can you approved this please?

File: files/jquery.inview-fork/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+{
+  "packageManager": "bower",
+  "name": "protonet/jquery.inview",
```
Comment (jimaek) [inline files/jquery.inview-fork/update.json]: The name is what is going to be used to search bower. `protonet/jquery.inview` is not a valid bower package name.
Either fix it or change to Github package manager

**Comment by sergeymakinen** [comment]
@jimaek could this PR be approved please?

**Comment by ricardopolo** [comment]
Why not merged?

**Pull request closed without merging**